src/pg/decoder.cr

```diff
@@ -408,23 +408,29 @@ module PG
     struct TimeDecoder
       include Decoder
 
-      DATE_OID = 1082
+      enum OID
+        DATE        = 1082
+        TIMESTAMP   = 1114
+        TIMESTAMPTZ = 1184
+      end
       JAN_1_2K = Time.utc(2000, 1, 1)
 
-      def_oids [
-        DATE_OID, # date
-        1114,     # timestamp
-        1184,     # timestamptz
-      ]
```

Comment:
This was just me cleaning up a bit, I figured OIDs were a fantastic use case for `enums.